src/screens/CreatePost/index.js

```diff
@@ -702,7 +702,7 @@ const CreatePost = () => {
     } else {
       setTypeUser(false);
     }
-  }, [params.followType]);
+  }, [params?.followType]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is good for preventing potential errors. By using optional chaining (`?.`), you're ensuring that if `params` is undefined, the code won't throw an error trying to access `followType`. Instead, it will return `undefined`, which is a much safer behavior.
```diff
-  }, [params.followType]);
+  }, [params?.followType]);
```
This change improves the robustness of your code by making it more resilient to unexpected situations where `params` might not be defined.


<!-- This is an auto-generated comment by OSS CodeRabbit -->